kops/kops.go

```diff
@@ -121,29 +94,25 @@ func NewKops() (*KopsCmd, error) {
 //}
 
 func (k *KopsCmd) ReplaceCluster(cluster clusteroperatorv1alpha1.ClusterSpec) error {
-	pwd, err := os.Getwd()
-	if err != nil {
-		return err
-	}
-
 	tempConfigFile := cluster.Name + ".yaml"
-	err = utils.CopyBufferContentsToTempFile([]byte(cluster.Config), tempConfigFile)
+	err := utils.CopyBufferContentsToTempFile([]byte(cluster.Config), tempConfigFile)
 	if err != nil {
 		return err
 	}
 
-	kopsCmdStr := "/usr/local/bin/" +
-		"docker run" +
```

Comment:
kops.go in master is no longer using the docker approach for kops and is instead using a binary. This makes the command structure a bit different, so when you merge with master all kops function command strings can be updated to call just the binary and not use any of this docker stuff!